### DISCUSSION TITLE
[Feature] Reactive Mongo not support,cause there is no Reative Mongo agent or plugins

### DISCUSSION THREAD

**Discussion by wild46cat** [apache/skywalking#12819] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

when I use Reactive Mogo ,the trace is not completed.
cause I can't found my Reactive Mongo span.

### Use case

support Reactive Mongo trace agent

### Are you willing to submit a pull request to implement this on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
As mentioned before, new plugin feature is only a discussion for Java agent. 
The maintainer will not own this, it is open to volunteers. The Java agent is super popular, and the maintainers can't afford to support one by one.

If you need more, submit pull requests or wait.

**Reply by wild46cat** [reply]
ok
